### DISCUSSION TITLE
cannot show line numbers while using Prsim.highlight

### DISCUSSION THREAD

**Discussion by jiawei686** [PrismJS/prism#2931] [Q&A] [answered]
I findout that prism cannot show line numbers while using Prsim.highlight. After I checked out line-numbers plugin's source code, I realized it's based on some DOM operate.
![image](https://user-images.githubusercontent.com/12953093/120739437-1367db00-c524-11eb-8d7c-f115edec6a91.png)
So it seems available while using Prism.highlightElement. Is there a way that I can show line numbers while using Prism.highlight?

**Comment by RunDevelopment** [comment, marked as answer]
No, this is not possible. Sorry.

Line Numbers needs full DOM access to implement features like soft wrapping and clickable line numbers. Its interactions with the Line Highlight plugin also require DOM access.

**Reply by jiawei686** [reply]
Got it. Thanks!
